commerce-product-type-virtual-order-service/src/main/java/com/liferay/commerce/product/type/virtual/order/service/http/CommerceVirtualOrderItemServiceHttp.java

```diff
@@ -69,12 +69,7 @@
 				returnObj = TunnelUtil.invoke(httpPrincipal, methodHandler);
 			}
 			catch (Exception e) {
-				if (e instanceof Exception) {
-					throw (Exception)e;
-				}
-
-				throw new com.liferay.portal.kernel.exception.SystemException(
-					e);
+				throw e;
```

Comment:
doesn't this mean we can just remove the `catch` block?

Reply:
Yes, and that we don't need the returnObj variable either